stdlib/public/core/CMakeLists.txt

```diff
@@ -220,8 +203,67 @@ add_swift_library(swiftCore ${SWIFT_STDLIB_LIBRARY_BUILD_TYPES} IS_STDLIB IS_STD
     section_magic
     ${GROUP_INFO_JSON_FILE}
   SWIFT_COMPILE_FLAGS ${swift_stdlib_compile_flags}
-  LINK_FLAGS ${swift_core_link_flags}
-  PRIVATE_LINK_LIBRARIES ${swift_core_private_link_libraries}
+  LINK_FLAGS ${SWIFT_RUNTIME_SWIFT_LINK_FLAGS} "-all_load"
   INCORPORATE_OBJECT_LIBRARIES swiftRuntime swiftStdlibStubs
-  FRAMEWORK_DEPENDS ${swift_core_framework_depends}
+  FRAMEWORK_DEPENDS Foundation CoreFoundation
+  TARGET_SDKS ALL_APPLE_PLATFORMS
   INSTALL_IN_COMPONENT stdlib)
+
+add_swift_library(swiftCore ${SWIFT_STDLIB_LIBRARY_BUILD_TYPES} IS_STDLIB IS_STDLIB_CORE
+  ${SWIFTLIB_SOURCES}
+  # The copy_shim_headers target dependency is required to let the
+  # build system know that there's a rule to produce the shims
+  # directory, but is not sufficient to cause the object file to be rebuilt
+  # when the shim header changes.  Therefore, we pass both the target
+  # and the generated directory as dependencies.
+  FILE_DEPENDS
+    copy_shim_headers "${SWIFTLIB_DIR}/shims"
+    section_magic
+    ${GROUP_INFO_JSON_FILE}
+  SWIFT_COMPILE_FLAGS ${swift_stdlib_compile_flags}
+  LINK_FLAGS ${SWIFT_RUNTIME_SWIFT_LINK_FLAGS}
+  PRIVATE_LINK_LIBRARIES "/Users/bgesiak/GitHub/apple/libiconv-libicu-android/armeabi-v7a/libicuuc.so" "/Users/bgesiak/GitHub/apple/libiconv-libicu-android/armeabi-v7a/libicui18n.so"
```

Comment:
Uhm... whoops! This obviously shouldn't be in here... 😅 

Removing now. I'll replace this with platform-specific libicu library paths.